src/com/connectsdk/service/capability/MediaControl.java

```diff
@@ -133,13 +133,42 @@ else if (transportState.equals("NO_MEDIA_PRESENT")) {
         }
     }
 
+    /**
+     * Get MediaControl implementation
+     * @return MediaControl
+     */
     public MediaControl getMediaControl();
+
+    /**
+     * Get a capability priority for current implementation
+     * @return CapabilityPriorityLevel
+     */
     public CapabilityPriorityLevel getMediaControlCapabilityLevel();
 
+    /**
+     * Plays the current media item
+     * @param listener (optional) ResponseListener< Object > with methods to be called on success or failure
```

Comment:
Strange, this `listener` param is only documented here, and not in `pause()`, `stop()`, etc. Either remove from here or add it to all the other methods.